valhalla/jawn/src/lib/handlers/LoggingHandler.ts

```diff
@@ -418,4 +458,63 @@ export class LoggingHandler extends AbstractLogHandler {
 
     return requestInsert;
   }
+
+  private pullRequestBodyMessage(requestBody: any): string {
+    try {
+      const messagesArray = requestBody.messages;
+
+      if (!Array.isArray(messagesArray)) {
+        return "";
+      }
+
+      const allMessages = messagesArray
+        .filter((message) => {
+          return message.role === "user";
+        })
```

Comment:
I don;t think we want to filter here at all. This will miss system or assistant message.